tests/automatic_tags_der.rs

```diff
@@ -0,0 +1,42 @@
+pub use asn1rs::prelude::*;
+
+asn_to_rust!(
+    r"Data-Structures DEFINITIONS AUTOMATIC TAGS ::=
```

Comment:
Do you have a plan to implement this `AUTOMATIC TAGS`?

Reply:
To be honest, I didn't look into the Tag stuff too much, since it was never relevant for uPER. I am open to it being implemented.